.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -56,8 +48,15 @@ body:
   - type: textarea
     id: logs
     attributes:
-      label: Browser console logs.
-      description: Check our [debugging](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html#debugging) FAQ section to provide this required information. Please attach INFO-level logs with timestamps to the report from *before* the session started up until the point at which the unexpected behavior occurred. For multi-participant calls, attach a log file for each participant if possible.
+      label: Browser console logs
+      description: |
+        Check our [debugging](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html#debugging) FAQ section to provide this required information.
+        Please attach INFO-level logs with timestamps to the report from *before* the session started up until the point at which the unexpected behavior occurred.
+        For multi-participant calls, attach a log file for each participant if possible.
       render: shell
```

Comment:
Do we want them to paste the console log here or do attachment?

Reply:
Good question. Got to know if we specify the `render` option then users cannot attach the files. Removed that option.

Reference link: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#textarea